xbmc/storage/windows/Win32StorageProvider.cpp

```diff
@@ -96,6 +98,7 @@ CDetectDisc::CDetectDisc(const std::string &strPath, const bool bautorun)
 bool CDetectDisc::DoWork()
 {
   CLog::Log(LOGDEBUG, "%s: Optical media found in drive %s", __FUNCTION__, m_strPath.c_str());
+#ifdef HAS_DVD_DRIVE
```

Comment:
this ifdef isn't needed because the ifdef above avoid running this code. if you want to keep this for safety, I think the log record above shouldn't be occurs also.